examples/nas/darts/retrain.py

```diff
@@ -4,24 +4,16 @@
 
 import torch
 import torch.nn as nn
-from nni.nas.pytorch.fixed import apply_fixed_architecture
-from nni.nas.pytorch.utils import AverageMeter
 from torch.utils.tensorboard import SummaryWriter
 
 import datasets
 import utils
 from model import CNN
+from nni.nas.pytorch.fixed import apply_fixed_architecture
+from nni.nas.pytorch.utils import AverageMeter
 
-logger = logging.getLogger()
-
-fmt = '[%(asctime)s] %(levelname)s (%(name)s/%(threadName)s) %(message)s'
-logging.Formatter.converter = time.localtime
-formatter = logging.Formatter(fmt, '%m/%d/%Y, %I:%M:%S %p')
+logger = logging.getLogger('nni')
```

Comment:
change another name?

Reply:
It's root logger of the code, and nni logger is initialized, so it set to nni logger here.